client/src/app/app.css

```diff
@@ -1,63 +1,64 @@
-.pf-c-icon {
-  --pf-c-icon--FontSize: var(--pf-global--FontSize--md);
-  --pf-c-icon--Color: #72767b;
+.pf-v5-c-icon {
+  --pf-v5-c-icon--FontSize: var(--pf-v5-global--FontSize--md);
+  --pf-v5-c-icon--Color: #72767b;
 
-  --pf-c-icon--m-sm--FontSize: var(--pf-global--icon--FontSize--sm);
-  --pf-c-icon--m-md--FontSize: var(--pf-global--icon--FontSize--md);
-  --pf-c-icon--m-lg--FontSize: var(--pf-global--icon--FontSize--lg);
-  --pf-c-icon--m-xl--FontSize: var(--pf-global--icon--FontSize--xl);
-  --pf-c-icon--m-2xl--FontSize: var(--pf-global--icon--FontSize--2xl);
+  --pf-v5-c-icon--m-sm--FontSize: var(--pf-v5-global--icon--FontSize--sm);
+  --pf-v5-c-icon--m-md--FontSize: var(--pf-v5-global--icon--FontSize--md);
+  --pf-v5-c-icon--m-lg--FontSize: var(--pf-v5-global--icon--FontSize--lg);
+  --pf-v5-c-icon--m-xl--FontSize: var(--pf-v5-global--icon--FontSize--xl);
+  --pf-v5-c-icon--m-2xl--FontSize: var(--pf-v5-global--icon--FontSize--2xl);
 
-  --pf-c-icon--m-info--Color: var(--pf-global--primary-color--100);
-  --pf-c-icon--m-success--Color: var(--pf-global--success-color--100);
-  --pf-c-icon--m-warning--Color: var(--pf-global--warning-color--100);
-  --pf-c-icon--m-danger--Color: var(--pf-global--danger-color--100);
+  --pf-v5-c-icon--m-info--Color: var(--pf-v5-global--primary-color--100);
+  --pf-v5-c-icon--m-success--Color: var(--pf-v5-global--success-color--100);
+  --pf-v5-c-icon--m-warning--Color: var(--pf-v5-global--warning-color--100);
+  --pf-v5-c-icon--m-danger--Color: var(--pf-v5-global--danger-color--100);
 
-  font-size: var(--pf-c-icon--FontSize);
-  color: var(--pf-c-icon--Color);
+  font-size: var(--pf-v5-c-icon--FontSize);
+  color: var(--pf-v5-c-icon--Color);
 }
-.pf-c-icon .pf-m-sm {
-  font-size: var(--pf-c-icon--m-sm--FontSize);
+.pf-v5-c-icon .pf-v5-m-sm {
+  font-size: var(--pf-v5-c-icon--m-sm--FontSize);
 }
 
-.pf-c-icon .pf-m-lg {
-  font-size: var(--pf-c-icon--m-lg--FontSize);
+.pf-v5-c-icon .pf-v5-m-lg {
+  font-size: var(--pf-v5-c-icon--m-lg--FontSize);
 }
 
-.pf-c-icon .pf-m-xl {
-  font-size: var(--pf-c-icon--m-xl--FontSize);
+.pf-v5-c-icon .pf-v5-m-xl {
+  font-size: var(--pf-v5-c-icon--m-xl--FontSize);
 }
 
-.pf-c-icon .pf-m-default {
-  color: var(--pf-c-icon--Color);
+.pf-v5-c-icon .pf-v5-m-default {
+  color: var(--pf-v5-c-icon--Color);
 }
 
-.pf-c-icon.pf-m-info {
-  color: var(--pf-c-icon--m-info--Color);
+.pf-v5-c-icon.pf-v5-m-info {
+  color: var(--pf-v5-c-icon--m-info--Color);
 }
 
-.pf-c-icon .pf-m-success {
-  color: var(--pf-c-icon--m-success--Color);
+.pf-v5-c-icon .pf-v5-m-success {
+  color: var(--pf-v5-c-icon--m-success--Color);
 }
 
-.pf-c-icon .pf-m-warning {
-  color: var(--pf-c-icon--m-warning--Color);
+.pf-v5-c-icon .pf-v5-m-warning {
+  color: var(--pf-v5-c-icon--m-warning--Color);
 }
 
-.pf-c-icon .pf-m-danger {
-  color: var(--pf-c-icon--m-danger--Color);
+.pf-v5-c-icon .pf-v5-m-danger {
+  color: var(--pf-v5-c-icon--m-danger--Color);
 }
```

Comment:
I'm surprised that this CSS was ever necessary. I wonder if we just remove all the `pf-v5-c-icon` stuff here if icons work as expected?

@gildub it looks like this was added in https://github.com/konveyor/tackle2-ui/pull/300/files#diff-acb6efd446c502574b8cfc71c2c9fda16948c4811f0040ed7efcbfd1ebdee22f, do you remember why it was needed?

Reply:
Thanks for point it that.
I'll check.

Reply:
@mturley, Effectively those classes seems unnecessary, I removed them. I can't remember why I added them in the first place.